### DISCUSSION TITLE
Failed : GetCertificates_Orchestrator

### DISCUSSION THREAD

**Discussion by ash06dec** [polymind-inc/acmebot#556] [Q&A] [answered]
Orchestrator function 'GetCertificates_Orchestrator' failed: The activity function 'GetAllCertificates' failed: "Retry failed after 4 tries. Retry settings can be adjusted in ClientOptions.Retry or by configuring a custom retry policy in ClientOptions.RetryPolicy. (An attempt was made to access a socket in a way forbidden by its access permissions. (asrwe-lowcode-s-kvt-sbx.vault.azure.net:443)) (An attempt was made to access a socket in a way forbidden by its access permissions. (asrwe-lowcode-s-kvt-sbx.vault.azure.net:443)) (An attempt was made to access a socket in a way forbidden by its access permissions. (asrwe-lowcode-s-kvt-sbx.vault.azure.net:443)) (An attempt was made to access a socket in a way forbidden by its access permissions. (asrwe-lowcode-s-kvt-sbx.vault.azure.net:443))". See the function execution logs for additional details.

![image](https://user-images.githubusercontent.com/17750641/216982000-403c30b7-3886-472f-b96d-49c433839807.png)

FYI,

Function app has all permission on the given KV.
Re-deployed couple of times.

Its been working in others env flawlessly but new deployment shows error. Any idea ?

**Comment by shibayan** [comment, marked as answer]
From the error message, it appears to be a network socket level error, so it appears to be either a temporary Azure outage or a problem with the Firewall configuration on the Key Vault side.
